src/features/embalm/stepContent/hooks/useCreateSarcophagus.ts

```diff
@@ -108,7 +108,12 @@ export function useCreateSarcophagus() {
 
   useEffect(() => {
     // TODO: compare with pending fees instead
-    setHasApproved(allowance !== undefined && BigNumber.from(allowance).gte(ethers.constants.MaxUint256.sub(ethers.utils.parseEther('100'))));
+    setHasApproved(
+      allowance !== undefined &&
+        BigNumber.from(allowance).gte(
+          ethers.constants.MaxUint256.sub(ethers.utils.parseEther('100'))
+        )
+    );
```

Comment:
this will have merge conflicts with the PR i just opened (just noting, dont need to change anything)